src/basic/cgroup-util.c

```diff
@@ -1749,7 +1750,10 @@ int cg_get_keyed_attribute(
                 p += strspn(p, NEWLINE);
         }
 
-        r = -ENXIO;
+        if (mode & CG_KEY_MODE_GRACEFUL)
+                goto done;
+        else
+                r = -ENXIO;
```

Comment:
doesn't really matter, but the "else" can just be removed. That way we save one line and a bit of indentation on this line...